services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -200,6 +201,29 @@ def get_delete_project_task(
 #
 
 
+def _get_service_start_lock_key(user_id: UserID, project_uuid: ProjectID) -> str:
```

Comment:
May I please ask to move this helper functions to another protected module and keep in ``projects_api`` only interfaced functions (i.e. not prefixed ``_``) e.g. ``add_project_node`` 

I suggest to create ``projects/_nodes_utils.py``


Thx a lot! :-)